spec/feature/post_ruby_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Renders the post show page', type: :feature do
+  before :each do
+    @user = User.create(Name: 'Ouail', Photo: 'https://www.linkedtour.com/images/lkavatar.png',
+                        Bio: 'I am Micronaut, studying full stack development', PostsCounter: 0)
+    @first_post = Post.create(Title: 'Microverse',
+                              Text: 'Microverse online teaching school for those who want to be developers',
+                              CommentsCounter: 2, LikesCounter: 10, author: @user)
+    @second_post = Post.create(Title: 'Microverse', Text: 'The same post like the first one', CommentsCounter: 0,
+                               LikesCounter: 5,
+                               author: @user)
+    2.times do |_i|
+      Comment.create(Text: 'Nice post!!', author_id: @user.id, post_id: @first_post.id)
+    end
+    visit "/users/#{@user.id}/posts/#{@first_post.id}"
+  end
+
+  scenario 'displays the post title' do
+    expect(page).to have_content(@first_post.Title)
+  end
+
+  scenario 'shows the person who wrote the post' do
+    expect(page).to have_content('Ouail')
+  end
+
+  it 'can see the username of each commentor.' do
```

Comment:
- [ ] Please change this file name to `post_show_spec.rb`, It will to understand what tests it contains.